### ISSUE TITLE
DHCP not working

### ISSUE THREAD

**Issue by bikram990** [angelnu/pod-gateway#13]
# Details

**What steps did you take and what happened:**

I installed pod-gateway via helm charts and pod-gateway came up successfully.

I installed a sample app (terminal) in the routed namespace and upon checking the logs of gateway_sidecar I found that the DHCP server in pod-gateway is not responding.

**What did you expect to happen:**

dnsmasq in pod-gateway should offer IPs to the pods in routed namespace

**Anything else you would like to add:**

I tried following steps in pod-gateway shell:
- tcpdump on pod-gateway shows that dhcp packets are reaching pod-gateway pod but not to `dnsmasq`
- `dnsmasq` is able to receive a UDP packet sent via `nmap` how ever `broadcast-dhcp-discover` script fails

Gluetun info:
- gluetun is able to connect to the vpn
- gluetun always falls back to 1.1.1.1 nameserver as it finds dns server is not responding (I think this is due to network policy)

**Additional Information:**

chart version: 6.0.0
vpn type: openvpn
cluster is based on k3s and flannel network
Downgrading to k8s chart version 5.6.2 works

I think it might be related to https://github.com/angelnu/pod-gateway/issues/10

**Comment by justSem** [comment]
Hi,

I saw you mentioned my issue.
In my case the DHCP packets weren't even reaching the pod-gateway which turned out to be a Cilium based issue to fix [CVE-2019-13119](https://cilium.io/blog/2019/07/01/cve-2019-13119/) but that was on a new cluster where I installed cilium in order to do some more troubleshooting.

My actual issue started on a K3S/Calico cluster where I haven't been able to figure out the root cause, but switching to RKE2 + Canal (Calico/Flannel) worked fine with Chart 6.0.0.
The only thing bringing my connection down on the pod-gateway was a healthcheck component making requests to ipinfo.io and exceeding their rate-limit, causing the VPN to think it was offline due to it not receiving an HTTP 200 response.
